assets/src/edit-story/components/panels/utils/getCommonValue.js

```diff
@@ -32,7 +37,7 @@
  */
 function getCommonValue(list, property) {
   const first = list[0][property];
-  const allMatch = list.every((el) => el[property] === first);
+  const allMatch = list.every((el) => equal(el[property], first));
```

Comment:
I'm usually really skeptical about deep equal. Plus, I think it's not necessarily what we need here. I did something to that extent for masks, but likely we want to run `getCommonValue` against the collection of all links, e.g. to update the icon for several links. But the whole link equality is very unlikely.

Reply:
I just re-read the spec and it seem like links are disabled completely during multiple selection so I'll just revert this

Reply:
Reverted